src/plugins/intel_gpu/src/graph/strided_slice.cpp

```diff
@@ -45,7 +45,7 @@ std::vector<layout> strided_slice_inst::calc_output_layouts(strided_slice_node c
         || (end_data.empty() && !constant_mem.count(2))
         || (strides_data.empty() && !constant_mem.count(3))) {
         auto out_shape = ov::PartialShape::dynamic(input0_layout.get_partial_shape().size());
-        return { layout{out_shape, input0_layout.data_type, format::get_default_format(out_shape.rank().get_length())} };
+        return { layout{input0_layout.get_partial_shape(), input0_layout.data_type, format::get_default_format(out_shape.rank().get_length())} };
```

Comment:
Is there any potential issue when input rank is not same with output rank?

Reply:
I think there is no issue during compiling of dynamic shape. I've just used input0 layout as a default.